frontend/integration-tests/protractor.conf.ts

```diff
@@ -17,6 +17,8 @@ import {
 const tap = !!process.env.TAP;
 
 export const BROWSER_TIMEOUT = 15000;
+export const SLOW_BACKEND = 1.0;
```

Comment:
I'd rather expose a single env var for specifying a timeout. Then you can set that locally if your environment is slow.

Reply:
Agreed, env var would be a better approach, no touching files. What would you suggest we externalize? `BROWSER_TIMEOUT` or `SLOW_BACKEND` or something else?

Reply:
I think something like `JASMINE_TIMEOUT` that we just use as the default timeout. This would allow to tighten up some of the timeouts for local development, but still set a larger value for CI where we want to avoid flakes.

Reply:
Implemented `$JASMINE_TIMEOUT` support, please take a look.